lsp/src/test/scala/org/ensime/lsp/rpc/MessageCompanionsSpec.scala

```diff
@@ -49,10 +51,13 @@ object MessageCompanionsSpec {
   final case class AddTransactionResponse(id: Int) extends Response
 
   object ResponsesFormats extends DefaultJsonProtocol with FamilyFormats {
+    override implicit def optionFormat[T: JsonFormat]: JsonFormat[Option[T]] =
+      super.optionFormat
+
     implicit val updateAccountCommand: RootJsonFormat[UpdateAccountCommand] =
       cachedImplicit
-    implicit val addTransactionCommand: RootJsonFormat[AddTransactionResponse] =
-      rootFormat(JsonUtils.wrapperFormat(AddTransactionResponse, _.id))
+    implicit val addTransactionCommand: JsonFormat[AddTransactionResponse] =
```

Comment:
Not `RootJsonFormat`? Or we are going to remove it anyway?

Reply:
It's not a root.

Reply:
:thinking: why?

Reply:
roots are only objects and arrays, but you're contramapping over something else here